java/dynamodb-tools/src/main/java/sleeper/dynamodb/tools/DynamoDBRecordBuilder.java

```diff
@@ -35,7 +35,11 @@ public DynamoDBRecordBuilder string(String key, String value) {
     }
 
     public DynamoDBRecordBuilder number(String key, Number value) {
-        return add(new Attribute(key, createNumberAttribute(value)));
+        if (null == value) {
+            return this;
```

Comment:
Could we test the case where an attribute gets set to one value and then unset by calling this with a null? I think it should actually delete the value from the map.